data/links.json

```diff
@@ -79,6 +79,11 @@
         "href": "https://segfault.csh.rit.edu/",
         "icon": "material:layers"
       }
+      {
+        "name": "Quotefault",
+        "href": "https://quotefault.csh.rit.edu/",
+        "icon": "material:quote"
```

Comment:
I am more partial to `foundation:comment-quotes`, I think it will look better with the rest of the icons.